.changeset/red-kids-look.md

```diff
@@ -1,5 +1,5 @@
 ---
-"remix": patch
+"@remix-run/dev": patch
```

Comment:
Changed this for visibility since it seems `remix`-only changesets don't get picked up by the bot?